spec/serializers/directory_item_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe DirectoryItemSerializer do
+
+  context "serializes additional user attributes when site setting is enabled" do
```

Comment:
I think we should add another test that expect some user information on a serialized `ItemUser`.